ArticleTemplates/assets/js/modules/quiz.js

```diff
@@ -132,20 +131,39 @@ define([
                 }
             },
 
-            getCorrectAnswers: function () {
+            getQuestionsAndAnswers: function () {
                 var i,
-                    answers = [],
-                    correctAnswers = document.querySelector('.quiz__correct-answers').innerHTML.split(','),
-                    correctAnswerArray,
-                    correctAnswerObj;
-                
-                for (i = 0; i < correctAnswers.length; i++) {
-                    correctAnswerObj = {};
-                    correctAnswerArray = correctAnswers[i].split(':')[1].split('-');
-                    correctAnswerObj.code = correctAnswerArray[0].trim().toUpperCase();
-                    correctAnswerObj.explanation = correctAnswerArray[1] || '';
-                    answers.push(correctAnswerObj);
-                } 
+                    key,
+                    answerMatch,
+                    question,
+                    answer,
+                    answers = {},
+                    correctAnswerWordList = document.querySelector('.quiz__correct-answers').innerHTML.split(' ');
```

Comment:
@gtrufitt yep unfortunately this information is embedded in the page as the contents of an HTML element, not ideal! JSON would be much better I agree, unfortunately it's out of the scope for this particular ticket (would require server side work)..but it's the direction i certainly hope to head.